imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -43,13 +50,17 @@ class UnitExplorer extends Component {
                 ))}
               </div>
             </div>
-            <Link to='/unit/new' className='link'>
-              <MenuItem className='no-shrink' innerDivStyle={{padding: 0}}>
-                <div className='tc bondi-blue fw5 bt b--moon-gray'>
-                  Add Unit
-                </div>
-              </MenuItem>
-            </Link>
+            {fabDescriptors.map((desc, ind) => (
```

Comment:
Using a map to iterate over an array of 1 is a bit confusing in this case. I know the original usage pattern you found this in includes it, but it doesn't make sense in this scenario, as there shouldn't be more than 1 FAB in any scenario.
You can skip the ".map" and just render the FAB directly... copy over the value from `fabDescriptors` to be used directly here